tests/test_strategy_storage.py

```diff
@@ -339,14 +339,14 @@ def storage_strategy_test5(area_test5, called):
     s.offers.post(area_test5.past_market.offers['id3'], area_test5.past_market.id)
     s.offers.post(area_test5.past_market.offers['id2'], area_test5.past_market.id)
     s.offers.sold_offer('id2', area_test5.past_market)
-    assert s.state.used_storage == 5
+    assert(isclose(s.state.used_storage, 5, rel_tol=1e-05))
```

Comment:
I would also be fine with changing the initial_soc and battery_capacity_kWh in order to not need `isclose` (this applies to the whole test suite in this file.

Reply:
I saw that you didnt change this. But ok. Leave it now.
But please delete the unneeded brackets. Thanks!

Reply:
Please look into the commit after your comment. It has already been incorporated. Thanks!

Reply:
Sorry! I is was in the wrong timezone ;-)